src/cli/utils/groups.go

```diff
@@ -101,11 +101,18 @@ func MakeGroupsOpts(cmd *cobra.Command) GroupsOpts {
 }
 
 // ValidateGroupsRestoreFlags checks common flags for correctness and interdependencies
-func ValidateGroupsRestoreFlags(backupID string, opts GroupsOpts) error {
+func ValidateGroupsRestoreFlags(backupID string, opts GroupsOpts, isRestore bool) error {
 	if len(backupID) == 0 {
 		return clues.New("a backup ID is required")
 	}
 
+	// The user has to explicitly specify which resource to restore. In
+	// this case, since we can only restore sites, the user is supposed
+	// to specify which site to restore.
+	if isRestore && len(opts.Site) == 0 {
+		return clues.New("a site ID or web URL is required")
```

Comment:
Our standard in sharepoint is to separate siteIDs from weburls with the --site-id and --site flags, respectively. Reducing it to --site works, but i'm a bit worried about confusing end users.  Mostly speaking out loud.  We should probably find a way to remove the --site-id flag for sharepoint to keep things uniform across service commands.

Reply:
Yeah, I think we could have just had a single flag. I don't wanna change that behaviour in this PR. I think that should be a separate larger change.